src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -44,14 +44,20 @@ export interface IChoiceGroupOption {
   imageSize?: { width: number, height: number };
 
   /**
-   * Whether the option is disabled or not.
+   * Whether the options is checked or not.
    * @defaultvalue false
    */
-  isDisabled?: boolean;
+  isChecked?: boolean;
 
   /**
-   * Whether the options is checked or not.
-   * @defaultvalue false
+   * Whether or not the option is disabled.
    */
-  isChecked?: boolean;
+  disabled?: boolean;
+
+  // @todo: Update version numbers for depriate and removal
+  /**
+   * Deprecated at v0.xx.0, to be removed at >= v0.xx.0. Use 'disabled' instead.
```

Comment:
If we know what release this will go out with, I can update that before merge (happens in 2 places)